ext/swiftlint/Rakefile

```diff
@@ -16,7 +16,7 @@ namespace :swiftlint do
     SWIFTLINT_MD5_HASH = DangerSwiftlint::SWIFTLINT_HASH
 
     puts "Downloading swiftlint@#{VERSION}"
-    sh "sh #{SCRIPT_PATH}/downloadSwiftlint.sh -u #{URL} -d #{DESTINATION} -a #{ASSET} -dh #{SWIFTLINT_MD5_HASH}"
+    sh "bash #{SCRIPT_PATH}/downloadSwiftlint.sh -u #{URL} -d #{DESTINATION} -a #{ASSET} -dh #{SWIFTLINT_MD5_HASH}"
```

Comment:
This makes sense, I originally had `./` instead of `sh` for the execution here but I changed it because I thought the problem was here, but it was a validation in our `danger-swiftlint.gemspec` in `spec.file` that was not allowing `.sh` files. You might also want to try `./` instead of `bash`. Because the script already has `#!/usr/bin/env bash` as shebang. I also found this as reference [ref](https://travis-ci.community/t/cant-run-bash-script-in-after-success-not-found/8669)

Reply:
Nice find. I'll try with `./`.

I also found this error, even though the CI build passed:
```
/home/travis/build/ashfurrow/danger-ruby-swiftlint/ext/swiftlint/downloadSwiftlint.sh: line 37: md5: command not found
342Zip was corrupted, try again later.
```
Keep in mind that the CI runs in linux. This is the config:
```
{
  "language": "ruby",
  "os": [
    "linux"
  ],
  "dist": "xenial"
}
```

Reply:
@dirtyhabits97 Thanks for taking care of the CI, I did not consider `linux` before, good catch 🚀